Sources/Controls/UIScrollView+Combine.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  UIScrollView+Combine.swift
+//  CombineCocoa
+//
+//  Created by Joan Disho on 09/08/2019.
+//  Copyright © 2019 Shai Mishali. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+public extension UIScrollView {
+    /// A publisher emitting content offset changes from this UIScrollView.
+    var contentOffsetPublisher: AnyPublisher<CGPoint, Never> {
+        publisher(for: \.contentOffset)
+            .eraseToAnyPublisher()
+    }
+
+    /// A publisher emitting if the bottom of the UIScrollView is reached.
+    ///
+    /// - parameter offset: A threshold indicating how close to the bottom of the UIScrollView this publisher should emit.
+    ///                     Default 0
+    /// - returns: AnyPublisher that emits when the bottom of the UIScrollView is reached.
```

Comment:
```suggestion
    /// - returns: A publisher that emits when the bottom of the UIScrollView is reached within the provided threshold.
```
no need to specify the concrete type itself